Application/src/visuals/ActorClass.java

```diff
@@ -0,0 +1,144 @@
+package visuals;
+
+import javafx.scene.canvas.GraphicsContext;
+import javafx.scene.image.Image;
+import javafx.scene.paint.Color;
+
+/**
+ * An ActorClass wrapper modeled after DiagramClass.
+ * @author Pontus Laestadius
+ * collaborator Kosara Golemshinska
+ * @version 1.0
```

Comment:
doesn't the versioning start with 0.1?

Reply:
good point, I will check with the rest of the code

Reply:
It starts at 1 when it gets put up for a pull request if I remember it correctly

Reply:
oh, okay, thanks